src/styles/_tags.scss

```diff
@@ -84,113 +85,113 @@
         }
     }
 
-    // Tagify and non-tagify paizo style main traits row
-    &.paizo-style {
-        border: none;
-        gap: 0;
-        padding: 0.5em 0.05em;
-
-        tag,
-        .tag,
-        select.tag {
-            background-color: var(--color-bg-trait);
-            border: solid var(--color-border-trait);
-            border-radius: 0;
-            box-shadow: none;
-            margin: 0;
+    &.light > .tag,
+    > .tag.light {
+        --tag-color: var(--color-pf-alternate-dark);
+        align-items: center;
+        background: none;
+        border-radius: 2px;
+        border: 1px solid var(--tag-color);
+        box-shadow: none;
+        color: var(--tag-color);
+        display: flex;
+        font: 700 var(--font-size-14) / 1.5em var(--sans-serif);
+        font-variant: all-small-caps;
+        letter-spacing: normal;
+        height: 1.5em;
+        margin: 0;
+        padding: 0 var(--space-4);
+        white-space: nowrap;
+        width: auto;
+    }
+}
 
-            x {
-                align-items: start;
-                margin: 0;
-            }
+// Tagify and non-tagify paizo style main traits row
+.tags.paizo-style {
```

Comment:
Unnesting these is introducing unnecessary risk of them getting forgotten when a nested block is updated

Reply:
I'd say its pretty necessary. Global styles need to use as few identifying tags as possible so that they can be overriden by specific uses of them. The only alternative is a mixin, or fully divorcing paizo style from non-paizo style stylings so that they're repeats and one does not affect the other.

This kinda changes it back to the original style of tags + paizo style specific overrides separately, but changes the original tags to use a less specific variant if its paizo-style.

Reply:
Now that I think more on it, I'm starting to lean towards divorcing the styles, though I'm hesistant to do that in a patch release. What I did here was effectively a soft revert, but I changed the main css class, fixed the actual source of the bug, and kept nesting the tag light into the main style.

Reply:
I noticed .tags.paizo-style in the main stuff was unnecessary though so I removed it. I don't know a good selector to divorce the styles while maintaining low specificity though. We'd need a divergent name from paizo style.